### ISSUE TITLE
parallel batch eats up a lot of memory

### ISSUE THREAD

**Issue by laurentopia** [cmdr2/stable-diffusion-ui#1237]
With the same settings, A1111 can handle it but this will go out of memory.
(I like this one better though so I stick around)

**Comment by JeLuF** [comment]
Did you change the memory profile in the settings tab to "low" instead of "balanced"?

**Comment by fdwr** [comment]
> (I like this one better though so I stick around)

@laurentopia: Yeah, having tried A1111 tonight, I definitely appreciate cmdr2's version with a streamlined enqueueing process, scrollable image queue, and a generally faster ability to follow down the random rabbit holes of specific variations.

**Comment by laurentopia** [comment]
> Did you change the memory profile in the settings tab to "low" instead of "balanced"?

That's what I was using.
